crates/amaru-kernel/src/memoized/transaction_output.rs

```diff
@@ -137,6 +139,50 @@ impl<'a> TryFrom<MintedTransactionOutput<'a>> for MemoizedTransactionOutput {
     }
 }
 
+// TODO: avoid clones (just getting the rebase to continue, will fix after rebase)
```

Comment:
@Jsh I understand you plan to address this as part of this PR?

Reply:
Yes, I totally forgot about this. Thanks!